### DISCUSSION TITLE
usethis::edit_r_profile() doesn't launch .Rprofile

### DISCUSSION THREAD

**Discussion by brndngrhm** [posit-dev/positron#4415] [Q&A] [answered]
typing `usethis::edit_r_profile()` doesn't open my Rprofile, instead it seems like its maybe waiting for me to do something then times out. I loved the feature in R studio to launch my Rprofile or R environment for quick edits, can this be incorporated into positron too?

![image](https://github.com/user-attachments/assets/98c0b62d-0cff-46c8-a665-16ad2b49b553)

**Comment by juliasilge** [comment, marked as answer]
Hmmmm, I can't reproduce this problem as `usethis::edit_r_profile()` works for me as expected. It looks like you are using or have in the past used the VSCode-R extension. That extension is not compatible with Positron, [as we outline here](https://github.com/posit-dev/positron/wiki#compatibility). Can you check if that extension is installed, or if you have code somewhere that may be calling that extension? Double check that any code you have in your `.Rprofile` that is designed to support that extension is updated so that it does not run in Positron:

```r
if (interactive() && Sys.getenv("RSTUDIO") == "" && Sys.getenv("POSITRON") == "") {
  ## code you use for better terminal behavior for R in VS Code
}
```

**Reply by brndngrhm** [reply]
thanks! my Rprofile had this in there, which i think was some config for VS code? probably something i set up when i started using VSCode and forgot about.

```
 if (interactive() && Sys.getenv("RSTUDIO") == "") {
   source(file.path(Sys.getenv(if (.Platform$OS.type == "windows") "USERPROFILE" else "HOME"), ".vscode-R", "init.R"))
 }
```
 I commented it out and restarted and now `usethis::edit_r_profile()` works in positron
